client/myReviews.html

```diff
@@ -23,17 +23,19 @@
   <script src="js/bootstrap/bootstrap.min.js"></script>
   <!-- Addons Javascripts -->
   <script src="js/bootstrap/modal.js"></script>
-  <!--<script src="javascripts/bootstrap/dropdown.js"></script>--> 
 
   <!-- leaflet JS -->
   <script src="http://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.js"></script>
   <script type='text/javascript' src='http://cdnjs.cloudflare.com/ajax/libs/leaflet.markercluster/0.4.0/leaflet.markercluster.js'></script>
 
   <!-- app Js -->
   <script src="js/utils.js"></script>
+  <script src="js/AJAXRequest.js"></script>
   <script src="js/restaurantsAPI.js"></script>
   <script src="js/connectionsAPI.js"></script>
   <script src="js/myReviews.js"></script>
+  <script src="js/clientLogic.js"></script>
+  <script src="js/drawModule.js"></script>
```

Comment:
if these scripts do not have any DOM dependency then it is better to load them with <script defer>